crates/matrix-sdk-ui/src/timeline/controller/state.rs

```diff
@@ -295,6 +300,19 @@ impl TimelineState {
         result
     }
 
+    pub(super) fn reload_shields(&mut self) {
+        let Ok(is_room_encrypted) = self.meta.is_room_encrypted.read() else {
```

Comment:
We generally `unwrap()` if a lock is poisoned, it's the only place where we allow unwraps.